blog/Website/2015-02-18_FreshRSS_gets_a_new_website.md

```diff
@@ -0,0 +1,6 @@
+Title: FreshRSS gets a new website
+Date: 2015-02-18
+
+We have worked a lot to improve community processes since last weeks. Today we are opening a new section on the website: a blog!
+
+TODO: this article needs to be finished!
```

Comment:
When you say we need to finish the article before putting it live. Do you mean this one?